src/view/configuration/configuration.jsx

```diff
@@ -168,6 +170,9 @@ const validateDuplicateValue = (createError, instances, key, message) => {
 const idSyncContainerIdValidationMessage =
   "Please specify a non-negative integer or data element for the container ID.";
 
+const onBeforeEventSendValidationMessage =
+  "Please specify a data element.";
```

Comment:
Will you please do the same for the all the other validation messages so that we're consistent in our approach? You can do it in a different PR if you'd like.